LICENSE

```diff
@@ -631,8 +631,8 @@ to attach them to the start of each source file to most effectively
 state the exclusion of warranty; and each file should have at least
 the "copyright" line and a pointer to where the full notice is found.
 
-    <one line to give the program's name and a brief idea of what it does.>
-    Copyright (C) <year>  <name of author>
+    {one line to give the program's name and a brief idea of what it does.}
+    Copyright (C) {year}  {name of author}
```

Comment:
What about this? Is it the year the project started? What about the author? Maybe the person who started the project plus mention that additional authors are in AUTHORS.md?

Reply:
[Here](https://www.gnu.org/licenses/gpl-howto.en.html) they say we should add year for each release and it's okay to use a range, plus we should mention all people who contributed code, so maybe:

*2008-2017, Nicolay Korslund and others (see AUTHORS.md)*?

Reply:
That's fine, I would just leave out the link to AUTHORS.md so the license file is valid on its own if someone forgets to package authors.md.

Reply:
"Nicolay Korslund and Contributors" maybe.